package.json

```diff
@@ -35,7 +35,8 @@
   },
   "devDependencies": {
     "ejs": "^2.4.1",
```

Comment:
You can drop the "ejs" dep now that gendocs.js is gone.

Reply:
Good call!